core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1291,7 +1291,7 @@ private void waitForTableStateTransition(TableId tableId, TableState expectedSta
           maxPerServer = serverCounts.max();
           waitTime = maxPerServer * 10;
         } else
-          waitTime = waitFor * 10;
+          waitTime = (long) waitFor * 10;
```

Comment:
Would it be cleaner to do: `10L * waitFor`? (Similar comment on similar changes in this PR.)

Reply:
This [stackoverflow answer](https://stackoverflow.com/questions/1494862/multiplying-long-values) explains why.

Reply:
> This [stackoverflow answer](https://stackoverflow.com/questions/1494862/multiplying-long-values) explains why.

Sorry, no... it doesn't. You didn't link to an answer... only a question. The accepted answer explains why what I wrote above would work fine... but it does not explain why a cast would be preferred over using the "L" like on my suggestion.